### PR TITLE
updated import and function id/hashes in readme

### PR DESCRIPTION
**Pull request by miiklay**
added missing FunctionRef import, and updated function ID/hashes for echo examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,11 +70,11 @@ Invoke is used to run a function repeatedly with multiple inputs. It gives you m
 Example [invoke_echo.py](https://github.com/capeprivacy/pycape/blob/main/examples/invoke_echo.py):
 
 ```py
-from pycape import Cape
+from pycape import Cape, FunctionRef
```
Comment (yanndupis) [inline README.md]: Nice catch for the missing import! 
`nit`: can you just have the import on separate lines? we try to follow [this guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#313-imports-formatting):
```
from pycape import Cape
from pycape import FunctionRef
```
Thank you

File: README.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ Example [run_echo.py](https://github.com/capeprivacy/pycape/blob/main/examples/r
 from pycape import Cape, FunctionRef
```
Comment (yanndupis) [inline README.md]: Same comment

**Review by yanndupis — COMMENTED** [review]
Thanks for opening this pr, I just left a small comment.

File: README.md (hunk 1 above)
Comment (jvmncs) [reply README.md:73]: @miiklay note this can be done automatically by running `make fmt` from the project root

File: README.md (hunk 2 above)
Comment (miiklay) [reply README.md]: agree. made the change

**Review by yanndupis — APPROVED** [review]
lgtm

**Pull request merged**